spyder/plugins/editor/widgets/editor.py

```diff
@@ -889,17 +889,17 @@ def set_bookmark(self, slot_num):
     def inspect_current_object(self):
         """Inspect current object in the Help plugin"""
         editor = self.get_current_editor()
-        editor.sig_display_signature.connect(self.display_signature_help)
+        editor.sig_display_object_info.connect(self.display_help)

```

Comment:
This may affect the Hover Hints PR

Reply:
I agree but the signal name was misleading.